src/app/views/realms/view/RouteRealmsView.vue

```diff
@@ -99,31 +56,36 @@ function onShareClick() {
 	}
 	ShareModal.show({ resource: 'realm', url: shareLink.value });
 }
-
-function onPostAdded(post: FiresidePost) {
-	ActivityFeedService.onPostAdded({
-		feed: feed.value!,
-		post,
-		appRoute: appRoute,
-		route: route,
-		router: router,
-	});
-}
 </script>
 
 <template>
-	<AppShellPageBackdrop v-if="realm">
+	<AppShellPageBackdrop v-if="routeStore.realm.value">
```

Comment:
After you destructure you won't need to use .value in the templates anymore

Reply:
also remove <AppTranslate>

Reply:
remove what?